src/python/deepgnn/graph_engine/snark/distributed.py

```diff
@@ -84,3 +92,12 @@ def reset(self):
         """Reset server."""
         if self.server is not None:
             self.server.reset()
+
+    def __reduce__(self):
+        """On serialize reload as Client."""
+
+        def deserialize(*args):
+            get_logger().setLevel(logging.ERROR)
+            return Client(*args)
+
+        return deserialize, ([self._hostname], self._ssl_cert)
```

Comment:
How do we handle a multi-server setup?

Reply:
This will have to be done manually e.g. in the transform function just manually create a client.

Reply:
Discuss above.